lib/cjs/src/components/homeComponents/Section.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Leaving a comment as to why you are exporting an empty object might be helping above this

Reply:
This is caused by an empty component file being compiled at build. The component has been destroyed.